auto_round/inference/backend.py

```diff
@@ -137,7 +137,7 @@ def feature_multiply_checker_group_size(in_feature, out_feature, group_size, in_
                                                   requirements=["auto-round>=0.5.0", "triton>=2.0"]
                                                   )
 
-BackendInfos['auto_round:tritonv2_zp'] = BackendInfo(device=["cuda"], sym=[True],  ## asym has accuracy issue
+BackendInfos['auto_round:int32_zp'] = BackendInfo(device=["cuda"], sym=[True],  ## asym has accuracy issue
```

Comment:
Keep the existing backend name unchanged and add a new backend named auto_round:torch. All AutoRound/GPTQ model packing should target the  qlinear_int32 or int32_zp, while inference should use either Triton or Torch, depending on the backend name or compatibility.